src/prop-types/tools/areComponentsEqual.js

```diff
@@ -0,0 +1,12 @@
+const RHL_PREFIX = '__reactstandin__getCurrent';
+
+const getProxyOrType = c => {
+  const isProxy = c && typeof c !== 'string' && Reflect.has(c, RHL_PREFIX);
```

Comment:
`Reflect` is not supported by IE11 - Any alternatives?

Reply:
Sure, will add fallback.